bpf/lib/maps.h

```diff
@@ -44,14 +44,20 @@ static __always_inline __must_check int
 tail_call_policy_dynamic(struct __ctx_buff *ctx, __u16 endpoint_id)
```

Comment:
Nit: I'd take the opportunity to rename this one to `tail_call_policy` to match `tail_call_egress_policy`. The 'dynamic' part is an implementation detail.

Reply:
Let's do that when we eliminate the `tail_call_policy_static()` variant? I'd prefer not to backport the rename :slightly_smiling_face:.

Reply:
I'm actually doubting if we should go through with unifying the static and dynamic helpers since it may result in subtle perf regressions. I'd rather eliminate all `tail_call_{static,dynamic}` calls in business logic and wrap them in helpers like you've done here.

Reply:
think I understand the idea, will follow up